shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -36,6 +36,7 @@
 
     public TextInputPlugin(FlutterView view) {
         mView = view;
+        mView.requestFocus();
```

Comment:
I think this should be done in `TextInputPlugin.showTextInput`.

Another view may have obtained focus between the construction of the `FlutterView` and the Flutter app's usage of the keyboard.